README.md

```diff
@@ -398,7 +398,7 @@ and add this config to your `package.json`:
   "lint-staged": {
     "*.{js,json,css}": [
       "prettier --write",
-      "git add"
+      "git add" // Warning: This breaks staging specific lines. See footnote below.
```

Comment:
Hmm, this is JSON, which doesn't support comments ...

Reply:
Well isn't this embarassing 😅